src/main/resources/config.yml

```diff
@@ -1,3 +1,186 @@
 message-on-login-enabled: true
```

Comment:
Maybe put an immutable revision at the top, so incase we add new entries or deprecate existing ones, we can do config upgrades based on the version the user has?

Reply:
This should also be set somewhere in the code, in case if someone removes or changes it.
Comment can be like `# The first line is immutable. Don't change the value, it'll reset itself after every reboot.`